src/elements/content-preview/ContentPreview.scss

```diff
@@ -75,3 +75,22 @@
 .bcpr.bcpr-thumbnails-open .bcpr-navigate-left {
     left: 226px;
 }
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .be.bcpr {
+            position: relative;
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
```

Comment:
question: Is this for modifying the position of the navigation buttons (`.bcpr-navigate-left` and `.bcpr-navigate-right`) to make them vertically centered with the screen as opposed to the preview container when the sidebar is on the bottom?

If not, what's the reason for this change?